reflex/app.py

```diff
@@ -112,11 +112,29 @@ def default_backend_exception_handler(exception: Exception) -> EventSpec:
         EventSpec: The window alert event.
 
     """
+    from reflex.components.sonner.toast import Toaster, toast
+
     error = traceback.format_exc()
 
     console.error(f"[Reflex Backend Exception]\n {error}\n")
 
-    return window_alert("An error occurred. See logs for details.")
+    error_message = (
+        [f"{type(exception).__name__}: {exception}.", "See logs for details."]
+        if get_config().debug is True
+        else ["Contact the website administrator."]
```

Comment:
Should we still mention that an error occured here

Reply:
It is still mentioned as the title of the toast.

There does seems to be a bug where the title disappear for some reason.